README.md

```diff
@@ -0,0 +1,48 @@
+# NYPL Header App
+The NYPL Header App can be used for projects across the NYPL organization to import the Header and Footer components.
```

Comment:
Add at the end, "...import the Header and Footer through embeddable Javascript scripts."

Reply:
Can you also add that the Header and Footer are built with the NYPL Reservoir Design System and are hosted in this repo as React components?